.github/workflows/rust.yml

```diff
@@ -134,3 +134,79 @@ jobs:
 
       - name: 🏃🏻‍♀️ Test
         run: CC=emcc AR=emar wasm-pack test --node --features wasm
+
+  benchmarks:
+    strategy:
+      fail-fast: false
```

Comment:
Should we set this to true? If an error is reported it'll be in the setup and install code as opposed to in the benchmarking itself, and it'd be better to just fail the whole run (if a failure is reported the whole thing has to be run anyway another time before it can be merged in).

Reply:
Good point, changed it